automation_tools/satellite6/upgrade/client.py

```diff
@@ -174,12 +174,12 @@ def satellite6_client_setup():
                 host=sat_host
             )
         # Refresh subscriptions on clients
-        time.sleep(5)
+        time.sleep(30)
```

Comment:
I understood the issue w/ yum lock. However do we really need 30 sec sleep ? Just asking if lowering down to 15secs would work ?